JumpscaleLibs/clients/stellar/StellarClient.py

```diff
@@ -0,0 +1,182 @@
+"""
+Stellar Client
+"""
+
+from Jumpscale import j
+try:
+    from stellar_sdk import Server, Keypair, TransactionBuilder, Network
+    from stellar_sdk.exceptions import BadRequestError
+except (ModuleNotFoundError, ImportError):
+    j.builders.runtime.python3.pip_package_install("stellar_sdk")
+
+try:
+    from stellar_base import Address
+except (ModuleNotFoundError, ImportError):
+    j.builders.runtime.python3.pip_package_install("stellar_base")
+
+from stellar_sdk import Server, Keypair, TransactionBuilder, Network
+from stellar_sdk.exceptions import BadRequestError
+from stellar_base import Address
+
+import requests
```

Comment:
`j.clients.http` should be the one used It's a wrapper around requests. If you don't use Session feature from requests, better to use it to stay in jumpscale context